src/main/metabase/api.ts

```diff
@@ -248,11 +264,15 @@ export async function buildInitialQueries(
   // Picks some fields which aren't AIDs to put in the GROUP BY.
   const nonAidField = fields.find((field) => !aidColumns.includes(field.name));
 
-  const columnSQL = nonAidField ? `"${nonAidField.name}",` : '';
-  const groupBySQL = nonAidField ? 'GROUP BY 1' : '';
-  const query = `SELECT ${columnSQL} count(*) FROM "${tableName}" ${groupBySQL}`;
+  const columnSQL = nonAidField ? `${postgresQuote(nonAidField.name)},` : '';
+  const groupBySQL = nonAidField ? `GROUP BY ${postgresQuote(nonAidField.name)}` : '';
+  const query = [`SELECT ${columnSQL} count(*)`, `FROM ${postgresQuote(tableName)}`, `${groupBySQL}`].join('\n');
+  const commentedQuery = `
+${query}
+
+${sqlHint}`;
```

Comment:
The following should better control where we want (and how many)  newlines:

```suggestion
  const commentedQuery = `${query}\n\n${sqlHint}`;
```